tabu/sampler.py

```diff
@@ -105,6 +106,9 @@ def sample(self, bqm, initial_states=None, initial_states_generator='random',
             num_restarts (int, optional, default=1,000,000):
                 Number of tabu search restarts per read.
 
+            energy_threshold (float, optional):
+                Terminate when this energy threshold is surpassed.
```

Comment:
I'd say something like ~ when energy _lower_ than `energy_threshold` is found ...